Sources/Rendering/Core/ImageMapper/Constants.d.ts

```diff
@@ -0,0 +1,14 @@
+export declare enum SlicingMode {
```

Comment:
`SlicingModes` if we want to be consistent.

Reply:
well it should SlicingModes but the Constants.js we have SlicingMode